pills/08-generic-builders.xml

```diff
@@ -49,7 +49,7 @@
 
     <screen><xi:include href="./08/hello-nix.txt" parse="text" /></screen>
     <note><title>Nix on darwin</title>
-    <para>The darwin (i.e. macOS) <literal>stdenv</literal> diverges from the Linux <literal>stdenv</literal> in several ways. A main difference is that the darwin <literal>stdenv</literal> relies upon <literal>clang</literal> rather than <literal>gcc</literal> as its C compiler. We can adapt this early example of how a <literal>stdenv</literal> works for darwin by using this modified version of <filename>hello.nix</filename>:
+    <para>Darwin (i.e. macOS) relies upon <literal>clang</literal> rather than <literal>gcc</literal> as its C compiler. We can adapt this early example for darwin by using this modified version of <filename>hello.nix</filename>.  Later we will show how nix can automatically handle these differences.
```

Comment:
This change makes it sound like this is about the platforms themselves while it is describing Nixpkgs’ `stdenv`. The whole pills are just “Build your own `stdenv`” guide so perhaps we could try to write around this without hiding the difference.

Reply:
@jtojnar sounds like we had the same conclusion? You like my fix?

Reply:
Yup, looks good.